src/components/Copybox.vue

```diff
@@ -39,7 +39,7 @@
 <style scoped>
   .copybox {
```

Comment:
Copy box CSS has some issues, when viewed on the 'Verify NAV address' page.

The label not longer 'floats' to the right position, it just jumps there. It also causes the input box to jump.

The box 'jumps' when you click inside it.

Reply:
fixed, by deleting margin-bottom: -23px; font-size: 15px;